fury/tests/test_actors.py

```diff
@@ -566,7 +566,7 @@ def test_peak_slicer(interactive=False):
 
 @npt.dec.skipif(skip_it or not have_dipy)
 @xvfb_it
-def test_tensor_slicer(interactive=False):
+def test_tensor_slicer(interactive=True):
```

Comment:
Can you put it back to `False`? The tests are not running because of that

Reply:
Sorry, my bad. I changed it for test purposes and left it behind. 69b395f reverts that change.